### PR TITLE
Validate the provided API version

### PR DESCRIPTION
**Pull request by Kemosabert**

### CODE REVIEW

File: src/utils/validateApiVersion.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export const validateApiVersion = version => {
```
Comment (duivvv) [inline src/utils/validateApiVersion.js]: PP: use export default here

File: test/utils/validateApiVersion.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  });
+
+  it('should recognize that the version date is in the past', () => {
+    expect(isVersionDateInTheFuture('2018-09-01')).toBe(false);
+  });
+
+  it('should recognize that the version date is in the future', () => {
+    expect(isVersionDateInTheFuture('2030-12-31')).toBe(true);
+  });
+});
```
Comment (duivvv) [inline test/utils/validateApiVersion.test.js]: Maybe we should also add a test for an invalid date? 15 as month, 34 as day, not sure how easy that would be without lib.

Comment (Kemosabert) [reply test/utils/validateApiVersion.test.js]: Hmm, actually I found out this about JS, I think it's a bit weird:
screen shot 2018-12-11 at 10 35 12

Comment (duivvv) [reply test/utils/validateApiVersion.test.js]: If it isn't trivial to do I would keep it like this for now.

**Comment by duivvv** [comment]
could you re-sync it with master?

File: src/utils/validateApiVersion.js (hunk 1 above)
Comment (duivvv) [reply src/utils/validateApiVersion.js]: @Kemosabert can you change this to keep it consistent with the rest of the modules in the repo?

**Comment by Kemosabert** [comment]
Closing this branch and PR and will open a new one

**Pull request closed without merging**